luigi/static/visualiser/js/visualiserApp.js

```diff
@@ -504,9 +504,6 @@ function visualiserApp(luigi) {
 
         $('input[name=vis-type]').on('change', function () {
             changeState('visType', $(this).val());
-
-            initVisualisation(visType);
```

Comment:
That would work but then `updateVisType` would be called twice since `processHashChange` also calls it.